### DISCUSSION TITLE
Add Creative Commons CC0-1.0 to License Analyzer?

### DISCUSSION THREAD

**Discussion by incredimike** [enlightn/enlightn#14] [Ideas]
I received the following warning for a CC0-1.0 license. This is the most permissive CC license, which puts works into the public domain without attribution required.

https://creativecommons.org/publicdomain/zero/1.0/

Perhaps this should be added to the License checks?

```
Check 52/60: Your application does not rely on dependencies you are not legally allowed to use. Failed
Your application has a total of 1 package(s) that you may not be legally allowed to use. By default, we assume the MIT, Apache-2.0, ISC, BSD Clause 2 & 3 and LGPL licenses to be legally valid for use for proprietary or commercial app
lications. However, you are free to change this in the Enlightn config. Unsafe packages include [simple-icons/simple-icons: CC0-1.0]
Documentation URL: https://www.laravel-enlightn.com/docs/security/license-analyzer.html
```

**Comment by paras-malhotra** [comment]
Hey @incredimike, thanks for the suggestion. I've added both CC0 and Unlicense to the whitelisted list. PR here: https://github.com/enlightn/enlightn/pull/15. This should be available in the next release.

**Reply by incredimike** [reply]
Thanks! Great work on an awesome tool
